roles/st2mistral/tasks/main.yml

```diff
@@ -31,6 +33,11 @@
   when:
     - st2mistral_version != "latest"
     - st2mistral_version != "present"
+  notify: enable mistral upgrade
+  tags: st2mistral
+
+- name: Enable mistral upgrade
+  meta: flush_handlers
```

Comment:
Flushing handlers in the middle is usually a bad/undesired practice and should be made in rare cases as possible.
I understand this comes from the `enable mistral upgrade` handler. We can skip the Ansible linter for that task.